packages/in_app_purchase/in_app_purchase_android/README.md

```diff
@@ -24,6 +24,6 @@ If you would like to contribute to the plugin, check out our
 [contribution guide](https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md).
 
 
-[1]: ../in_app_purchase/in_app_purchase
+[1]: https://github.com/flutter/plugins/tree/master/packages/in_app_purchase
```

Comment:
@ryoheiudagawa1995, I think it is better to point to the repository of `in_app_purchase` package it self (which would be `https://github.com/flutter/plugins/tree/master/packages/in_app_purchase/in_app_purchase`, since that is wat the text is also referring to (to the `in_app_purchase` package). Thinking about it, it would make even more sense to point to the package on pub.dev as that is where most users would retrieve it from. 

Should have thought about that the first time (sorry) but maybe you can do another update and change it to:
```suggestion
[1]: https://pub.dev/packages/in_app_purchase
```